CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ _This release is scheduled to be released on 2021-01-01._
 - JSON Parse translation files with comments crashing UI. (#2149)
 - Calendar parsing where RRULE bug returns wrong date, add Windows timezone name support. (#2145, #2151)
 - Wrong node-ical version installed (package.json) requested version. (#2153)
+- Fix calendar fetcher subsequent timing (#2160)
```

Comment:
:+1:

Reply:
sorry, change that should not have made it in, will fix